buffer/src/main/java/io/netty/buffer/PooledByteBuf.java

```diff
@@ -138,7 +138,7 @@ public final ByteBuf unwrap() {
 
     @Override
     public final ByteBuf retainedDuplicate() {
-        return PooledDuplicatedByteBuf.newInstance(this, readerIndex(), writerIndex());
+        return PooledDuplicatedByteBuf.newInstance(this, this, readerIndex(), writerIndex());
```

Comment:
`PooledByteBuf` itself does not support wrapping (see `unwrap()` above is `final` and always returns `null`). So the `wrapped` and `unwrapped` parameters are only meaningful if the buffer can be wrapped.